.github/workflows/e2e-tests-main-devnet.yml

```diff
@@ -80,6 +78,11 @@ jobs:
           export RUSTC_WRAPPER=$HOME/.local/bin/sccache
           cargo build --release
 
+      - name: Build runtime
+        run: |
+          export RUSTC_WRAPPER=$HOME/.local/bin/sccache
+          cargo build --release -p aleph-runtime
```

Comment:
Huh, this really builds the wasm runtime? Reading the `Cargo.toml` I would not have expected that – am I missing something?

Reply:
Hmm...
by `-p aleph-runtime` we tell cargo to build only the package `aleph-runtime` and I guess is essentially the runtime, isn't it?

here you have the artifacts, including the output of this command: https://github.com/Cardinal-Cryptography/aleph-node/actions/runs/1583175148, seems reasonable in size

Reply:
oh, I get it now; please look at `build.rs`